workload/workload.py

```diff
@@ -364,6 +364,17 @@ def get_cancelled_task_graphs(self) -> Sequence[TaskGraph]:
                 )
         return cancelled_task_graphs
 
+    def calculate_slo(self) -> float:
```

Comment:
I think this should be configurable. Add a command line option to specify the ramp up and ramp down clips (default = 0). In the future we can add an `auto` option that does this based on `rho` calculation.